src/routes/v2/auth.js

```diff
@@ -75,7 +77,14 @@ class AuthRouter {
   async login(req, res) {
     const { email: rawEmail } = req.body
     const email = rawEmail.toLowerCase()
-    await this.authService.sendOtp(email)
+    try {
+      await this.authService.sendOtp(email)
+    } catch (err) {
+      // Log, but don't return so responses are indistinguishable
+      logger.log(
+        `Error occurred when attempting to login user ${email}: ${err}`
```

Comment:
we should probably have structured logs so it makes querying easier in the future. 

also, could we use `logger.error` here?

Reply:
Swapped to logger.error in https://github.com/isomerpages/isomercms-backend/pull/624/commits/e2f81734a80b758ec845f84403c7457850aaceaa!

Currently our logger is initialised with no params, and our custom error middleware adds in the timestamp for each message - what kind of format were you thinking of adding?

Reply:
was hoping for a object of something like
```ts
message: string,  
meta: Record<string, int>
timestamp: number
``` 

or something